NEWS.md

```diff
@@ -21,7 +22,7 @@ The new features include:
 
 ## Minor changes
 
-* The column name for both `languagecodes` and `countrycodes` is now `code`, and not `alpha2` and `Code`, respectively. 
+* The column name for `countrycodes` is now `code`, not `Code`. 
```

Comment:
I'm wondering whether such changes should be "minor breaking changes" but this is a minor comment.

Reply:
Would you consider the second point a "minor breaking change" as well (I wouldn't, I think)? Or would that be two sections then, "minor breaking changes" and "minor changes"?

Reply:
Yeah maybe?

Reply:
How about making this a non-minor change and adding a `name` column to make it easier to find the right `countrycode`? From e.g. https://datahub.io/core/country-list#data